odc/stats/plugins/l34_utils/l4_water.py

```diff
@@ -5,43 +5,59 @@
 
 def water_classification(xx, intertidal_mask, water_persistence):
 
+    # Replace nan with nodata
```

Comment:
there is no need for intertidal_mask, intertidal in `classes_l3_l4` already.

Reply:
intertidal mask is removed.